server/src/main/java/com/cloud/configuration/Config.java

```diff
@@ -861,31 +861,31 @@
     VmDiskThrottlingIopsReadRate(
             "Advanced",
             ManagementServer.class,
-            Integer.class,
```

Comment:
Not sure why it was set as Integer. All the pieces of code using these configs are parsing them as Long.